### PR TITLE
[Feature Request] Carbon Black Response - Set Fetch Limit

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20865

## Description
Added *Max Incidents Per Fetch* field - Limit incidents created per fetch (default: 10).
Tested on the customer environment.

## Screenshots
image

## Does it break backward compatibility?
   - No: If value is null will fetch 10 incidents, so shouldn't alter behavior unless the user edits this field.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [Integrations/integration-Carbon_Black_Enterprise_Response.yml](https://github.com/demisto/content/edit/cbr-incr-fetch-limit/Integrations/integration-Carbon_Black_Enterprise_Response.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F5238)
- [ ] [Integrations/integration-Carbon_Black_Enterprise_Response_CHANGELOG.md](https://github.com/demisto/content/edit/cbr-incr-fetch-limit/Integrations/integration-Carbon_Black_Enterprise_Response_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5238)

### CODE REVIEW

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml [hunk 1]
```diff
@@ -43,6 +43,11 @@ configuration:
   defaultvalue: ""
   type: 0
   required: false
+- display: Max Incidents Per Fetch
```
Comment (ronykoz) [inline Integrations/integration-Carbon_Black_Enterprise_Response.yml]: I think that it should be something like `The maximum number of incidents to fetch` but need to consult with @kirbles19 about this
FYI build this example using https://confluence.paloaltonetworks.com/display/DemistoContent/Integrations+And+Scripts

**Comment by ronykoz** [comment]
@kirbles19 could you go over this change please?

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml (hunk 1 above)
Comment (kirbles19) [reply Integrations/integration-Carbon_Black_Enterprise_Response.yml]: Maximum number of incidents to fetch
OR
Maximum number of incidents per fetch

**Comment by kirbles19** [comment]
@DeanArbel Done. See my suggestions.

**Comment by DeanArbel** [comment]
@kirbles19 thank you. I updated thee display of the new parameter in accordance to your review

**Pull request merged**